src/pages/product/[productCode].tsx

```diff
@@ -67,6 +68,32 @@ export async function getStaticProps(
   const { locale, params } = context
   const { productCode } = params as any
   const product = await getProduct(productCode)
+  const relatedProducts = []
+  const relatedProductData = product?.properties?.find(
+    (item: any) => item.attributeFQN === 'tenant~related-products'
+  )?.values[0].stringValue
```

Comment:
what if values is null? add proper conditions

Reply:
changes done